### PR TITLE
readme: Add installation for Ubuntu and jenkins URL enhancement

### PR DESCRIPTION
**Pull request by aabawer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,14 @@ user:
 
     pip install -r requirements.txt --user
 
+On Ubuntu:
+
+    sudo apt install gir1.2-nm-1.0
+
```
Comment (nirs) [inline README.md:23]: This should come after the "On Fedora 28 or later" section.

The section about pip install should come last.

**Review by nirs — CHANGES_REQUESTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (aabawer) [reply README.md:23]: pip is about installing the gobject in case fedora dnf fails, not sure its the same pkg as in ubuntu

Comment (nirs) [reply README.md:23]: It is, pip install same packages on all platforms.

Comment (nirs) [reply README.md:23]: According to https://github.com/nirs/oci/blob/master/.travis.yml, we need more packages on Ubuntu. Did you test this on real Ubuntu installation?

Comment (aabawer) [reply README.md:23]: i meant: the pkg in requirements.txt is the python equiv of python2-gobject, not gir1.2-nm-1.0.

Comment (nirs) [reply README.md:23]: Looking in .travis.yaml, we need to run both dnf/apt and pip to get the pacakges, or we need to list python{2,3}-gobject in the distribution packages.

Comment (aabawer) [reply README.md:23]: there is a  python-gobject-2 for ubuntu which is the same as the one in python. 
but this not what solves the issue for ubuntu:

`Selecting previously unselected package python-gobject-2.

(Reading database ... 218679 files and directories currently installed.)
Preparing to unpack .../python-gobject-2_2.28.6-12ubuntu3_amd64.deb ...
Unpacking python-gobject-2 (2.28.6-12ubuntu3) ...
Setting up python-gobject-2 (2.28.6-12ubuntu3) ...
Processing triggers for libc-bin (2.29-0ubuntu2) ...
amit@spyron:~/git/my_oci$ ./ovirt-ci  system-tests 128938
Traceback (most recent call last):
  File "./ovirt-ci", line 2, in <module>
    from oci import main
  File "/home/amit/git/my_oci/oci/main.py", line 6, in <module>
    from . import gerrit
  File "/home/amit/git/my_oci/oci/gerrit.py", line 8, in <module>
    from . import network
  File "/home/amit/git/my_oci/oci/network.py", line 7, in <module>
    gi.require_version('NM', '1.0')
  File "/usr/lib/python2.7/dist-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace NM not available`

Comment (nirs) [reply README.md:23]: You are missing network manager gobject bindings, which probably come from gir1.2-networkmanager-1.0.

And this issue is probably the same for Fedora. So the correct way to list the packages for each distribution, and remove the pip install part. Mixing pip and system packages is not a good idea.

**Comment by nirs** [comment]
The ubuntu instructions added by 6047ec9908c5a143eca808c338933fada77d1483. Please check if this works for you.

**Pull request closed without merging**